docs/table-search-trends.md

```diff
@@ -73,6 +73,7 @@ Regional CSV files are available for download in our [data exploration and downl
 | Name | Type | Description | Example |
 | ---- | ---- | ----------- | ------- |
 | **key** | `string` | Unique string identifying the region | US_CA |
+| **place_ID** | `string` | A textual identifier that uniquely identifies a place in the Google Places database and on Google Maps ([details](https://developers.google.com/places/web-service/place-id))| ChIJd_Y0eVIvkIARuQyDN0F1LBA |
```

Comment:
Even though the Place ID is available at the source of data, users won't see it in our outputs for this table. Instead, PlaceID will be added to our index.
```suggestion
```